server/src/utils/utils.js

```diff
@@ -0,0 +1,26 @@
+const jsonwebtoken = require('jsonwebtoken');
+const fs = require('fs');
+const path = require('path');
+
+const pathToKey = path.join(__dirname, '..', 'id_rsa_priv.pem');
+const PRIV_KEY = fs.readFileSync(pathToKey, 'utf8');
+/**
+ * @param {*} user - The user object.  We need this to set the JWT `sub` payload property to the MongoDB user ID
```

Comment:
What is it for?

Reply:
It is reading that file that includes private key, if i remove that PRIV KEY the returned in token string , it throws this error "error": "secret must be a string or bufferor a KeyObject"